components/ui/Typography.jsx

```diff
@@ -1,18 +1,16 @@
-import classNames from "classnames";
+export const Heading = ({ children, className, type = "default" }) => {
+
+  const classes = {
+    one:`font-medium md:font-normal md:text-14.75 text-7.25 md:leading-22 leading-11 -tracking-0.02 ${className}`,
+    two:`font-normal md:text-11.75 md:leading-14.75 leading-7.15 text-6.5 text-white -tracking-0.015 md:-tracking-0.02 ${className}`,
+    three:`font-normal md:text-9.5 md:leading-12.25 leading-9 text-5.75 -tracking-0.015 ${className}`,
+    four: `font-normal leading-7.5 md:leading-9.5 text-xl md:text-3xl -tracking-0.015 ${className}`,
+    default:`font-normal text-white text-6.5 md:text-11.75 ${className}`
```

Comment:
this is not what I meant exactly. I was suggesting that one of the values would be the default one. 


If I was to use a  switch case I would something like 

```js
switch (type) {
    case 'two':
        return 'related classes';
    case 'three':
        return 'related classes';
    case 'four':
        return 'related classes';
    default:
        return 'classes to ONE';
}

```

In this case I used a switch case but you can use something different you can even set the value of type to `one` by default.

What I wanted to show here is that one of the types can be defined as the default one (fallback when type is not defined)

> If you have a default style which is different from any of these. you can ignore my comments above.